docs/multi-cluster-service.md

```diff
@@ -19,7 +19,7 @@ Ensure that at least two clusters have been added to Karmada, and the container
 
 We need to install ServiceExport and ServiceImport in the member clusters.
 
-After ServiceExport and ServiceImport have been installed on the **karmada control-plane**, we can create `ClusterPropagationPolicy` to propagate those two CRDs to the member clusters.
+After ServiceExport and ServiceImport have been installed on the Karmada Control Plane, we can create `ClusterPropagationPolicy` to propagate those two CRDs to the member clusters.
```

Comment:
We have bolded these keywords, why should we change them?

Reply:
I noticed that in README, 'Karmada Control Plane' was used, so I changed it to the same as README

Reply:
Thanks for your explanation, but I think that the focus may be different in different contexts, and I think it can be kept inconsistent.

Reply:
Okay, can I add the bold?